examples/braket_features/Noise_models/Noise_models_on_Rigetti_AspenM1.ipynb

```diff
@@ -0,0 +1,1058 @@
+{
```

Comment:
<p>nit: should we having a different variable and should there be a in-line comment to explain what are we doing here? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/152/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>